src/linter.ts

```diff
@@ -0,0 +1,69 @@
+import * as core from '@actions/core'
+import { readFile } from 'fs/promises'
+import type { LinterResponse } from './types'
+import fetch from 'node-fetch'
+
+export async function lintFiles(
+  files: string[],
+  apiKey: string,
+  axeLinterUrl: string,
+  linterConfig: Record<string, unknown>
```

Comment:
Why is this `unknown`? Shouldn't we know the config type here?

Reply:
If we're not going to strongly type our code, why use TypeScript?

Reply:
I think it's reasonable/good for the action to treat the config as opaque/unknown **iff the following are all true**:

- The action doesn't need to actually use the config (if it needs just parts, I'm fine with typing/validating just those parts)
- The service API it calls is responsible for parsing/validating the config
- When the service API detects an invalid config, it emits an actionable error response which the action displays reasonably to the user

Reply:
I'm with Michael on this. This code doesn't use the types at all. If it was easy to get the types in I'd say just do it anyway, but there isn't. We don't have the types published publicly. We'd either have to maintain a duplicate of a type that we don't use, or we'd have to add a publication step for an unused type. There's no value in that.